src/wireviz/Harness.py

```diff
@@ -97,8 +99,17 @@ def create_graph(self) -> Graph:
                      html_line_breaks(connector.subtype),
                      f'{connector.pincount}-pin' if connector.show_pincount else None,
                      connector.color, '<!-- colorbar -->' if connector.color else None],
-                    '<!-- connector table -->' if connector.style != 'simple' else None,
-                    [html_line_breaks(connector.notes)]]
+                    '<!-- connector table -->' if connector.style != 'simple' else None]
+            if connector.additional_components:
+                rows.append(["Additional components"])
+                for extra in connector.additional_components:
+                    qty = extra.qty * connector.get_qty_multiplier(extra.qty_multiplier)
+                    if(self.mini_bom_mode):
```

Comment:
The usage of parentheses here is very unusual in Python. Maybe you're also a C/C++ programmer as I am? 😄 I suggest removing these parentheses from the `if`-statement.